client-app/src/admin/tests/asyncLoops/AsyncLoopPanel.js

```diff
@@ -20,7 +20,7 @@ import {SECONDS} from '@xh/hoist/utils/datetime';
  * Test to demonstrate use of {@see whileAsync} and its more common cousin {@see forEachAsync}.
  */
 export const asyncLoopPanel = hoistCmp.factory({
-    model: creates(() => new Model()),
+    model: creates(() => AsyncLoopPanelModel),
```

Comment:
Any reason this can't be:

`creates(AsyncLoopPanelModel)`

Reply:
Yes, need closure, because not defined until further in this file.